ACL token read CLI command panics if both -self and -expanded flags are specified

#### Overview of the Issue

Running `consul acl token read` with both the `-self` and `-expanded` flags together causes a runtime panic.

Ideally, `-self` and `-expanded` should be possible to use together. In the meantime, Consul should report an error if both flags are specified (rather than experiencing a panic).

#### Reproduction Steps

Steps to reproduce this issue:

1. Run Consul 1.12.0+ with the ACL system bootstrapped
2. Set the bootstrap token to the `CONSUL_HTTP_TOKEN` env var
3. Run `consul acl token read -self -expanded
4. Receive an error output similar to this:

```
consul acl token read -expanded -self
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xa0 pc=0x1094f3d]

goroutine 1 [running]:
github.com/hashicorp/consul/command/acl/token.(*prettyFormatter).FormatTokenExpanded(0xc000361bac, 0x0)
	/home/runner/work/consul-enterprise/consul-enterprise/command/acl/token/formatter.go:120 +0x5d
github.com/hashicorp/consul/command/acl/token/read.(*cmd).Run(0xc000f025a0, {0xc00006e220?, 0xffffffffffffffff?, 0x0?})
	/home/runner/work/consul-enterprise/consul-enterprise/command/acl/token/read/token_read.go:111 +0x325
github.com/mitchellh/cli.(*CLI).Run(0xc000b7c640)
	/home/runner/go/pkg/mod/github.com/mitchellh/cli@v1.1.0/cli.go:260 +0x5f8
main.realMain()
	/home/runner/work/consul-enterprise/consul-enterprise/main.go:53 +0x40b
main.main()
	/home/runner/work/consul-enterprise/consul-enterprise/main.go:23 +0x19
```

### Affected versions

Consul 1.12.0+

--- a/.changelog/13787.txt
+++ b/.changelog/13787.txt
@@ -1,0 +1,3 @@
+```release-note:bug
+cli: when `acl token read` is used with the `-self` and `-expanded` flags, return an error instead of panicking
+```

--- a/command/acl/token/read/token_read.go
+++ b/command/acl/token/read/token_read.go
@@ -92,6 +92,13 @@ func (c *cmd) Run(args []string) int {
 			return 1
 		}
 	} else {
+		// TODO: consider updating this CLI command and underlying HTTP API endpoint
+		// to support expanded read of a "self" token, which is a much better user workflow.
+		if c.expanded {
+			c.UI.Error("Cannot use both -expanded and -self. Instead, use -expanded and -id=<accessor id>.")
+			return 1
+		}
+
 		t, _, err = client.ACL().TokenReadSelf(nil)
 		if err != nil {
 			c.UI.Error(fmt.Sprintf("Error reading token: %v", err))

